src/daemon/daemon.cpp

```diff
@@ -124,7 +124,7 @@ bool t_daemon::run(bool interactive)
       return false;
     mp_internals->rpc.run();
 
-    daemonize::t_command_server* rpc_commands;
+    daemonize::t_command_server* rpc_commands = NULL;
```

Comment:
Do I really have to do something silly like: rpc_commands = std::unique_ptr(new ...).release(); or is there simpler ? The docs don't show a ctor from the inner type (ie, rpc_commands = new ...;)